apps/SageAccountWeb/package.json

```diff
@@ -41,12 +42,14 @@
     "@types/react-tooltip": "^4.2.4",
     "@vitest/coverage-v8": "^0.34.6",
     "@vitest/ui": "^0.34.7",
+    "@vitest/utils": "^0.34.7",
     "assert": "^2.1.0",
     "buffer": "^6.0.3",
     "https-browserify": "^1.0.0",
     "identity-obj-proxy": "^3.0.0",
     "jsdom": "^21.1.2",
     "memfs": "^3.5.3",
+    "msw": "^2.3.1",
```

Comment:
Note: using MSW v2 which makes it much easier to write handlers. Other packages are using MSW v1.